services/galley/src/Galley/Data/Conversation.hs

```diff
@@ -82,20 +84,22 @@ convAccessData :: Conversation -> ConversationAccessData
 convAccessData conv =
   ConversationAccessData
     (Set.fromList (convAccess conv))
-    (convAccessRole conv)
+    (convAccessRoles conv)
 
-defRole :: AccessRole
-defRole = ActivatedAccessRole
+defRole :: Set AccessRoleV2
+defRole = activatedAccessRole
 
-maybeRole :: ConvType -> Maybe AccessRole -> AccessRole
-maybeRole SelfConv _ = privateRole
-maybeRole ConnectConv _ = privateRole
-maybeRole One2OneConv _ = privateRole
+maybeRole :: ConvType -> Maybe (Set AccessRoleV2) -> Set AccessRoleV2
+maybeRole SelfConv _ = privateAccessRole
+maybeRole ConnectConv _ = privateAccessRole
+maybeRole One2OneConv _ = privateAccessRole
 maybeRole RegularConv Nothing = defRole
 maybeRole RegularConv (Just r) = r
```

Comment:
- remove `defRole`
- rename `maybeRole` to `defRole` and move it to `libs/wire-api/src/Wire/API/Conversation.hs`
- then you don't need to export `{private,team,activated,nonActivated}AccessRole` from there.
- you will need to call the new `defRole` in a few places where you now use `{private,team,activated,nonActivated}AccessRole`, but I think that's ok?

to be more precise:

```suggestion
maybeRole :: ConvType -> Maybe (Set AccessRoleV2) -> Set AccessRoleV2
maybeRole = fromMaybe . defRole

defRole :: ConvType -> Set AccessRoleV2
defRole SelfConv = privateAccessRole
defRole ConnectConv = privateAccessRole
defRole One2OneConv = privateAccessRole
defRole RegularConv = activatedAccessRole
```